src/components/header.js

```diff
@@ -1,42 +1,117 @@
-import { Link } from "gatsby"
-import PropTypes from "prop-types"
-import React from "react"
-
-const Header = ({ siteTitle }) => (
-  <header
-    style={{
-      background: `rebeccapurple`,
-      marginBottom: `1.45rem`,
-    }}
-  >
-    <div
-      style={{
-        margin: `0 auto`,
-        maxWidth: 960,
-        padding: `1.45rem 1.0875rem`,
-      }}
-    >
-      <h1 style={{ margin: 0 }}>
-        <Link
-          to="/"
-          style={{
-            color: `white`,
-            textDecoration: `none`,
-          }}
-        >
-          {siteTitle}
-        </Link>
-      </h1>
-    </div>
-  </header>
-)
-
-Header.propTypes = {
-  siteTitle: PropTypes.string,
+import React from 'react'
+import { useStaticQuery, graphql } from 'gatsby'
+import Img from 'gatsby-image'
+import styled from "styled-components"
+
+import Link from "./link"
+import Socials from './socials'
+
+const NavLinkContainer = styled.li`
+  /* Items are centered in drop down */
+  align-self: center;
+  text-transform: uppercase;
+  font-weight: bold;
+`
+
+const NavLink = styled(Link)`
+/* Nav links should be white and then green when hovered over */
+color: var(--MHP-white) !important; 
+&:hover {
+  color: var(--MHP-green) !important; 
 }
+`
+
+const Navbar = styled.nav`
+padding-right: 0px;
+padding-top: 0px;
+padding-bottom: 0px;
+`
+
+const ColapsingDiv = styled.div`
+  justify-content: space-between;
+`
```

Comment:
No indentation vs 2 space indentation vs 4 space indentation is very triggering lmaoo

Reply:
Just ran a formatter and it fixed it!